frontend/packages/ceph-storage-plugin/src/plugin.ts

```diff
@@ -164,6 +165,28 @@ const plugin: Plugin<ConsumedExtensions> = [
       query: `${CAPACITY_USAGE_QUERIES[StorageDashboardQuery.CEPH_CAPACITY_USED]}[60m:5m]`,
     },
   },
+  {
+    type: 'Dashboards/Storage/TopConsumers/Item',
```

Comment:
this should go to kubevirt plugin as you want VMs to show up only when kubevirt is installed